playbooks/collect_instance_statistics/collect_elasticsearch_data.yml

```diff
@@ -19,6 +19,11 @@
         src: logs_server_requirements.txt
         dest: /tmp/logs_server_requirements.txt
 
+    - name: Copy Elasticsearch CA certificate
```

Comment:
@mtyaka, see [this comment](https://github.com/open-craft/ansible-secrets/pull/295/files#r685361497).